src/highlight.rs

```diff
@@ -46,7 +48,7 @@ impl Boundary {
     }
 }
 
-fn replace_tabs(s: &str, tab_len: u8) -> Cow<'_, str> {
+fn replace_tabs(s: &str, tab_len: u8, mask: bool) -> Cow<'_, str> {
```

Comment:
Can this be an enum instead of a bool?
https://rust-lang.github.io/api-guidelines/type-safety.html#c-custom-type